platforms/corda-ent/charts/bridge/values.yaml

```diff
@@ -0,0 +1,119 @@
+# Default values for bridge.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+deployment:
+  annotations: {}
+
+# Provide the name of the node.
+# e.g. nodeName: bridge
+nodeName: bridge
+
+# Provide information regarding the CENM Networkmap.
+# Used to curl the Networkmap service.
+nmap:
+  # Provide the name of the Networkmap deployment.
+  # e.g. name: nmap
+  name: nmap
+  # Provide the port where the Networkmap service is exposed.
+  # e.g. port: 10000
+  port: 10000
+  # Provide the namespace where the Networkmap is deployed.
+  namespace: baah
+
+# Provide context for the chart.
+metadata:
+  # Provide the namespace where the Bridge will be deployed.
+  # e.g. namespace: cenm
+  namespace: baah
+  # Provide any extra labels you wish to add to the charts, formatted as key-value pairs.
+  # e.g. key: "value" 
+  labels: {}
+
+# Provide the amount of replicas you want to deploy of the Float.
+# e.g. replicas: 1
+replicas: 1
+
+
+image:
+  initContainerName: index.docker.io/hyperledgerlabs/alpine-utils:1.0
+  imagepullsecret: regcred
+
+# Information about the Docker container used for the init-containers.
+initContainerImage:
+  # Provide the name of the image, including the tag.
+  # e.g. name: index.docker.io/hyperledgerlabs/alpine-utils:1.0
+  name:
+
+# Information about the main image used for the main Float firewall container.
+image:
+  # Provide the name of the image, including the tag
+  # e.g. name: adopblockchaincloud0502.azurecr.io/corda_image_firewall_4.4:latest
+  name:
+  # Provide the K8s secret that has rights for pulling the image off the registry.
+  # NOTE: Make sure the secret exists prior to manually running this deployment, 
+  # as this chart does not create the pull secret itself.
+  # e.g. pullSecret: regcred
+  pullSecret: 
+  # Provide the pull policy for Docker images, either Always or IfNotPresent.
+  # e.g. pullPolicy: Always
+  pullPolicy:
+
+
+# Information about the vault.
+# NOTE: Make sure that the vault is already unsealed, initialized and configured
+# to use the Kubernetes service account token based authentication.
+vault:
+  # Provide the vault address
+  # Eg. address: http://vault.example.com:8200
+  address: http://vault.internal.dev.aws.blockchaincloudpoc-develop.com:8200
```

Comment:
actual values need to be removed

Reply:
Fixed in [ec7e5a7](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/897/commits/ec7e5a75829cd0887cdedb55ee0c60e19266150b)